server/cluster/unsafe_recovery_controller_test.go

```diff
@@ -516,6 +517,8 @@ func (s *testUnsafeRecoverSuite) TestReportCollection(c *C) {
 func (s *testUnsafeRecoverSuite) TestPlanExecution(c *C) {
 	_, opt, _ := newTestScheduleConfig()
 	cluster := newTestRaftCluster(s.ctx, mockid.NewIDAllocator(), opt, core.NewStorage(kv.NewMemoryKV()), core.NewBasicCluster())
+	// Manually fill the coordinator up to allow calling on cluster.PauseOrResumeSchedulers().
+	cluster.coordinator = newCoordinator(s.ctx, cluster, hbstream.NewTestHeartbeatStreams(s.ctx, cluster.getClusterID(), cluster, true))
```

Comment:
It will be better if you checker the scheduler is pause  status.

Reply:
Done